### PR TITLE
updated the faq line spacing in account faq and updated the tips for vendors

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
updated the faq line spacing in account faq and updated the tips for vendors on how to request SEED provisioning.

### CODE REVIEW

File: docs/whats-new/production-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Invite and Get user APIs does not return any value for UserPrincipalName</summary>
+
+On a rare occasion, Azure may take up more time than expected to generate a user resource when invite user API is triggered. On such occasions, Invite and Get user APIs may return no value for **UserPrincipalName** .
+
+A fix has been applied to manage the delay from Azure and to return a valid error message when UserPrincipalName is empty for the following APIs.  
+
+**Invite user API**
+[Invite Public Officer](https://stg.docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1namespace~1{namespace}~1users~1publicofficer/post)
```
Comment (JLeeChan) [inline docs/whats-new/production-release-notes.md:46]: these four URLs should point to the production DocPortal (i.e. https://docs.developer.tech.gov.sg)

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -1,16 +1,39 @@
 # Staging release notes
 
+## Staging release 31 August 2022
+Frontend version: <version number in the environment> | Backend version: <version number in the environment>
+
+**New features** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Terminate accounts that are inactive for 30 days from creation</summary>
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md:9]: terminating accounts that are inactive for 30 days is not the new feature, the new feature is actually sending the notification emails 7 days prior to termination until the actual termination, so a total of 7 days

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -1,16 +1,39 @@
...
+
+<kbd>![email](../assets/images/whats-new/terminate-inactive-account.png)</kbd>
+</details>
+
+**Fixes** - **TechPass portal**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Warning message for expiring and expired certificates and secrets</summary>
+
+we have fixed a bug and now warning messages will be displayed for both expiring and expired certificates and secrets on the portal. Earlier, it was displayed only for expiring certificates and secrets.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md:21]: capital W to start the sentence

File: docs/whats-new/staging-release-notes.md [hunk 4]
```diff
@@ -1,16 +1,39 @@
...
+
+</details>
+
 ## Staging release 17 August 2022
-Frontend version:  | Backend version:   
+Frontend version: 1.0.0-20220808.0908 | Backend version: 1.27.8-220817.0220
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md:26]: i already see this info in https://docs.developer.tech.gov.sg/docs/techpass-user-guide/whats-new/staging-release-notes?id=staging-release-17-august-2022

you didn't pull the latest?

File: docs/whats-new/whats-new.md [hunk 5]
```diff
@@ -5,6 +5,7 @@ This page summarises the new features, improvements and bug fixes completed in e
 ## Latest changes
 
 **Production release notes**
+- [Production release 31 August 2022](whats-new/staging-release-notes?id=production-release-31-august-2022)
```
Comment (JLeeChan) [inline docs/whats-new/whats-new.md:8]: these 2 lines should point to **whats-new/production-release-notes**

File: docs/whats-new/staging-release-notes.md (hunk 4 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/staging-release-notes.md:26]: i think i pulled the latest but agreed i could see the version on the staging docs but my code didn't have that info, so i just copied the info from the published content.

File: docs/whats-new/whats-new.md (hunk 5 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/whats-new.md:8]: I think i have updated.. one link was pointing to staging in the prod release notes.

File: docs/whats-new/staging-release-notes.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/staging-release-notes.md:21]: done.

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/staging-release-notes.md:9]: got it, but the email sent to user does not clearly state that info.

File: docs/whats-new/production-release-notes.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/production-release-notes.md:46]: fixed, thanks!

**Review by JLeeChan — APPROVED** [review]

**Review by yqyeoh — APPROVED** [review]

**Pull request merged**